squid-blockexplorer/.gitignore

```diff
@@ -0,0 +1,8 @@
+/node_modules
+/lib
+
+/**Versions.json
+/**Versions.jsonl
```

Comment:
what's `.jsonl`?

Reply:
not sure, this is also from the original template [code](https://github.com/subsquid/squid-template/blob/main/.gitignore)